.github/workflows/ci.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           - '21'
         os:
           - windows-latest
-          - ubuntu-latest
+          - ubuntu-22.04
```

Comment:
Changed the version here, since this is the job that also tests Scala Native

Reply:
no need to do the same for windows-2022?

Reply:
ach yes, done